src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+            this._items.indexOf(t)
+          );
+        }),
+        (n._getItemByDirection = function (t, e) {
+          var n = "next" === t,
+            i = "prev" === t,
+            o = this._getItemIndex(e),
+            r = this._items.length - 1;
+          if (((i && 0 === o) || (n && o === r)) && !this._config.wrap)
+            return e;
+          var s = (o + ("prev" === t ? -1 : 1)) % this._items.length;
+          return -1 === s
+            ? this._items[this._items.length - 1]
+            : this._items[s];
+        }),
+        (n._triggerSlideEvent = function (t, n) {
+          var i = this._getItemIndex(t),
+            o = this._getItemIndex(
+              this._element.querySelector(".active.carousel-item")
+            ),
+            r = e.Event("slide.bs.carousel", {
+              relatedTarget: t,
+              direction: n,
+              from: o,
+              to: i,
+            });
+          return e(this._element).trigger(r), r;
+        }),
+        (n._setActiveIndicatorElement = function (t) {
+          if (this._indicatorsElement) {
+            var n = [].slice.call(
+              this._indicatorsElement.querySelectorAll(".active")
+            );
+            e(n).removeClass("active");
+            var i = this._indicatorsElement.children[this._getItemIndex(t)];
+            i && e(i).addClass("active");
+          }
+        }),
+        (n._slide = function (t, n) {
+          var i,
+            o,
+            r,
+            s = this,
+            a = this._element.querySelector(".active.carousel-item"),
+            c = this._getItemIndex(a),
+            u = n || (a && this._getItemByDirection(t, a)),
+            h = this._getItemIndex(u),
+            f = Boolean(this._interval);
+          if (
+            ("next" === t
+              ? ((i = "carousel-item-left"),
+                (o = "carousel-item-next"),
+                (r = "left"))
+              : ((i = "carousel-item-right"),
+                (o = "carousel-item-prev"),
+                (r = "right")),
+            u && e(u).hasClass("active"))
+          )
+            this._isSliding = !1;
+          else if (
+            !this._triggerSlideEvent(u, r).isDefaultPrevented() &&
+            a &&
+            u
+          ) {
+            (this._isSliding = !0),
+              f && this.pause(),
+              this._setActiveIndicatorElement(u);
+            var d = e.Event("slid.bs.carousel", {
+              relatedTarget: u,
+              direction: r,
+              from: c,
+              to: h,
+            });
+            if (e(this._element).hasClass("slide")) {
+              e(u).addClass(o), l.reflow(u), e(a).addClass(i), e(u).addClass(i);
+              var p = parseInt(u.getAttribute("data-interval"), 10);
+              p
+                ? ((this._config.defaultInterval =
+                    this._config.defaultInterval || this._config.interval),
+                  (this._config.interval = p))
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)